pkg/sql/row/row_converter.go

```diff
@@ -385,11 +385,18 @@ func (c *DatumRowConverter) Row(ctx context.Context, sourceID int32, rowIndex in
 	for i := range c.cols {
 		col := &c.cols[i]
 		if _, ok := c.IsTargetCol[i]; !ok && !col.Hidden && col.DefaultExpr != nil {
-			datum, err := c.defaultExprs[i].Eval(c.EvalCtx)
-			if err != nil {
-				return errors.Wrapf(err, "error evaluating default expression for IMPORT INTO")
+			if col.DefaultExprStr() == "unique_rowid()" {
+				if c.hidden < 0 {
```

Comment:
I don't think this is quite right? Not all tables into which we'll use `IMPORT INTO` with `unique_rowid()` will also be using _hidding_ PK cols, right?

also, I think this would currently be putting the same value in the hidden pk col and in the explicit default col, but that seems wrong -- each call to unique_rowid is supposed to return a different value, even if they're in the same row. Hmm.

Reply:
Ah, thanks for catching that. Turns out I omitted the case where `Primary Key` is specified in the column specification. 

For the second part of your comment, this would be a problem when we have more than a column with default expression being `unique_rowid()` right? Would it make sense to add a constraint that we only support if there's <= 1 column with `unique_rowid`?

Reply:
Huh, I thought I left a comment here... but I guess GitHub swallowed it?

Changing the method to `(uint64(sourceID) << rowIDBits) ^ uint64(c * rowIndex + r)` would work, but since the rows in a CSV import also add the timestamp to their rowID, it would also increase the rate at which we consume time. Right now, every row consumes 10 microseconds of time - with the modified expression, it would consume c*10 microseconds. Extrapolating from the comment in the code:

> 100m row file would use 10µs/row or ~17min worth of IDs

Then importing a (10 column x 100m) row file would use nearly 3 hours of IDs. That seems quite expensive... Perhaps we can keep track of only the columns which have a default value of `unique_rowid`. This would reduce the rate at which we consume keys to u*10 microseconds/row, where `u` is the number of columns with a `uniquie_rowid` default value.

But perhaps limiting 1 default `unique_rowid` per import is reasonable.

Reply:
Right, this concern makes sense. Maybe roll back to supporting only 1 default column with `unique_rowid` until we have a better scheme for avoiding collision?

Reply:
I'd have a slight preference for counting how many we need as previously suggested, instead of total columns, since a very wide table is pretty common but a wide table of uniquely-generated-otherwise-meaningless integers is not. In practice, I'd guess it is one most often (pk/hidden pk) but wouldn't be surprised by two: implicit pk and then explicit unique rowid (where they probably should have used that as an explicit PK instead but didn't)